src/components/profile.js

```diff
@@ -0,0 +1,136 @@
+import React, {Component} from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  Image,
+  Modal,
+  TouchableHighlight,
+} from 'react-native';
+import Icon from 'react-native-vector-icons/SimpleLineIcons';
+import AsyncStorage from '@react-native-community/async-storage';
+import {Actions} from 'react-native-router-flux';
+import ImagePicker from 'react-native-image-picker';
+
+import Styles from '../style/profile';
+import I18n from '../services/i18n';
+import api from '../services/api';
+
+export default class Profile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: null,
+      modalVisible: false,
+      errorMessage: '',
+    };
+  }
+
+  componentDidMount() {
+    this.getInfos();
+  }
+
+  async getInfos() {
+    const user = await AsyncStorage.getItem('@user');
+    const {avatar, email, nickname} = JSON.parse(user);
+    this.setState({avatar, email, nickname});
+  }
+
+  removeAccount = async () => {
+    try {
+      const token = await AsyncStorage.getItem('@token');
+      await api.delete('/v1/user', {
+        headers: `Bearer ${token}`,
+      });
+      Actions.Login();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://app.codacy.com/gh/Full-Moon-Castle/wizard/pullRequest?prid=5422182)